utils/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "celo-kona-utils"
```

Comment:
I wouldn't call this celo-kona-utils since the functionality seems to be focussed on open telemetry integration. Maybe `celo-otel` ?